plugin-templates/entries/slotfills/subheadline/Subheadline.tsx

```diff
@@ -0,0 +1,39 @@
+/* global tinyMCEPreInit */
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { Editor } from '@tinymce/tinymce-react';
+import { usePostMetaValue } from '@alleyinteractive/block-editor-tools';
+
+declare global {
+  const tinyMCEPreInit: {
+    baseURL: string,
+  };
+}
+
+function Subheadline() {
+  const [subheadline, setSubheadline] = usePostMetaValue('create_wordpress_plugin_subheadline');
+  return (
+    <PluginDocumentSettingPanel
+      // @ts-ignore
+      name="subheadline"
+      title={__('Subheadline', 'create-wordpress-project')}
+    >
+      <Editor
+        value={subheadline}
+        tinymceScriptSrc={`${tinyMCEPreInit.baseURL}/tinymce.min.js`}
```

Comment:
What if the tinyMCE API changes in the tinyMCEPreInit? You could check for the baseURL before outputting the `<Editor>` component.